databases/week2/databases-week2-Homework/school_databse.sql

```diff
@@ -0,0 +1,27 @@
+SET NAMES utf8mb4;
+
+-- tables
+
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`begins` DATETIME NULL DEFAULT NULL,
+`ends` DATETIME NULL DEFAULT NULL
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `student`(
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`email` varchar(255) NOT NULL,
+`phone` varchar(255) NULL,
+`class_id` int(10) unsigned NOT NULL,
+CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class`(`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- create index on name column from student table
+CREATE INDEX student_name_index
+ON student (name);
+
+-- Add column to table
+
+ALTER TABLE class ADD COLUMN status ENUM('not started', 'ongoing', 'finished')
```

Comment:
What would happen when the student has more than one class?
The answer is that you need a table that connects the student to classes.
maybe a student_classes table with studen_id and class_id as foreign keys.

Reply:
ok